### PR TITLE
EDB*Plus: v41.1.0 release

### PR DESCRIPTION
**Pull request by ccestes**
## What Changed?

Changes as part of the v41.1.0 release are as follows:

- [ED-244](https://enterprisedb.atlassian.net/browse/ED-244)
- [ED-246](https://enterprisedb.atlassian.net/browse/ED-246)

[ED-244]: https://enterprisedb.atlassian.net/browse/ED-244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ED-246]: https://enterprisedb.atlassian.net/browse/ED-246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: product_docs/docs/edb_plus/41/06_command_summary.mdx [hunk 1]
```diff
@@ -118,7 +115,7 @@ If the `COLUMN` command is followed by a column name, then the column name can b
 
 -  No other options
 -  `CLEAR`
--  Any combination of `FORMAT`, `HEADING`, and either `OFF` or `ON`
+-  Any combination of `FORMAT`, `HEADING,NEW_VALUE` and either `OFF` or `ON`
 
 `column`
```
Comment (mlewandowski-edb) [inline product_docs/docs/edb_plus/41/06_command_summary.mdx:120]: The `column` parameter/option was changed to `column_name` above.  Given that, either the subsequent references to the `column` parameter for the COLUMN command should be updated to refer to the parameter as `column_name` or `column_name` changed back to `column`.

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
See comment about updates needed for `column` versus `column_name` parameter for COLUMN command

File: product_docs/docs/edb_plus/41/06_command_summary.mdx (hunk 1 above)
Comment (ccestes) [reply product_docs/docs/edb_plus/41/06_command_summary.mdx]: I just made this update in my most recent commit. Please double-check that I changed the correct references.

**Review by mlewandowski-edb — APPROVED** [review]

**Review by mlewandowski-edb — APPROVED** [review]

File: product_docs/docs/edb_plus/41/02_release_notes/index.mdx
Comment (drothery-edb) [inline product_docs/docs/edb_plus/41/02_release_notes/index.mdx:1]: Need to add the nav order

File: product_docs/docs/edb_plus/41/06_command_summary.mdx [hunk 2]
```diff
@@ -148,13 +145,43 @@ If the `COLUMN` command is followed by a column name, then the column name can b
...
+`heading_text`
 
- Text to use for the column heading of `column`.
+ Text to use for the column heading of `column_name`.
+
+`NEW_V[ALUE]` 
+
+This variable option is valid if specified alone or with other already supported options. A variable specified in the `COLUMN` command option is associated with the given column. The variable can contain the data value of a column or expression of a variable in the SQL `SELECT` list and can be used in the SQL script for the remainder of the session.  
+
+To contain the `departmentnumber` value for the `deptno` column, a variable can be defined using `NEW_VALUE` option like follows:
```
Comment (drothery-edb) [inline product_docs/docs/edb_plus/41/06_command_summary.mdx:156]:
```suggestion
To contain the `departmentnumber` value for the `deptno` column, a variable can be defined using the `NEW_VALUE` option:
```

File: product_docs/docs/edb_plus/41/06_command_summary.mdx [hunk 3]
```diff
@@ -148,13 +145,43 @@ If the `COLUMN` command is followed by a column name, then the column name can b
...
+SQL> prompt departmentnumber: '&departmentnumber'
+departmentnumber: '40'
+SQL> INSERT INTO emp VALUES (1,'micheal','tester',7902,'17-DEC-80',800,NULL,'&departmentnumber');
+
+1 row INSERTED.
+```
+
+The `NEW_V[ALUE]` variable has the following limitations: 
+
+- Column values differ when they are seeded via the `NEW_VALUE` parameter for the "time with time zone" data type.
```
Comment (drothery-edb) [inline product_docs/docs/edb_plus/41/06_command_summary.mdx:178]:
```suggestion
- Column values differ when they are seeded using the `NEW_VALUE` parameter for the "time with time zone" data type.
```

File: product_docs/docs/edb_plus/41/06_command_summary.mdx [hunk 4]
```diff
@@ -148,13 +145,43 @@ If the `COLUMN` command is followed by a column name, then the column name can b
...
+- The `NEW_VALUE` variable prompts for the value if the corresponding column value is `NULL`. Oracle treats `NULL` and an empty string the same while EDB Postgres Advanced Server has different behavior. Therefore, in EDB*Plus it prompts for the value.
```
Comment (drothery-edb) [inline product_docs/docs/edb_plus/41/06_command_summary.mdx:179]:
```suggestion
- The `NEW_VALUE` variable prompts for the value if the corresponding column value is `NULL`. Oracle treats `NULL` and an empty string the same while EDB Postgres Advanced Server has a different behavior. Therefore, in EDB*Plus it prompts for the value.
```

**Pull request merged**